sklearn/tests/test_docstring_parameters.py

```diff
@@ -72,7 +71,8 @@ def test_docstring_parameters():
         assert sys.version_info >= (3, 5)
     except (ImportError, AssertionError):
         raise SkipTest(
-            "numpydoc is required to test the docstrings")
+            """numpydoc is required to test the docstrings,
```

Comment:
This puts a newline and whitespace in the error message. I think what you mean is:
```
(
    "Blah "
    "Continuation"
)
```